src/frontend/styles/_buttons.scss

```diff
@@ -1,22 +1,30 @@
+@mixin buttonDefaultStyle {
```

Comment:
not sure this changes are related to this PR, the sidebar uses link tags, wondering if these changes were committed by mistake?

Reply:
the sidebar links are <button> and <a> elements. iirc the <button> elements were using the primary font instead of the secondary font (hard to tell unless you change the fonts to look very different). I went ahead and made a mixin so that we could share properties between `<button>` and `.button` css classes because sharing all the properties breaks other <button> elements.

Reply:
oh weird, I checked a few of them and thought they were all `a` tags. I wonder why Manage Accounts is a button when other items that change the page are `a` tags :S.

I imagine the Discord/Patreo/Ko-fi are buttons because they open external urls.

Would be nice to standardize what we use there, but I guess that's a different thing to refactor haha.

Reply:
Yeah they should probably be standardized but I agree that should be a separate pr haha